client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/components/special-instructions.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Icon } from '@wordpress/components';
+import { chevronDown, chevronUp } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { Controller, Control } from 'react-hook-form';
+import FormTextArea from 'calypso/components/forms/form-textarea';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { CredentialsFormData } from '../types';
+import { ErrorMessage } from './error-message';
+
+interface Props {
+	control: Control< CredentialsFormData >;
+	errors: any;
+}
+
+export const SpecialInstructions: React.FC< Props > = ( { control, errors } ) => {
+	const translate = useTranslate();
+	const [ showNotes, setShowNotes ] = useState( false );
+
+	const toggleShowNotes = () => {
+		setShowNotes( ! showNotes );
+		recordTracksEvent( 'calypso_site_migration_special_instructions_toggle', {
+			show_notes: ! showNotes,
+		} );
+	};
+
+	return (
+		<div className="site-migration-credentials__special-instructions">
+			<Button onClick={ () => toggleShowNotes() } data-testid="special-instructions">
```

Comment:
Suggestion: Remove the tested and use `getByRole('button', {name: 'Special instructions'})` on our test suite.
It ensures our tests are checking if our component is accessible.

Reply:
Addressed the comments in this PR https://github.com/Automattic/wp-calypso/pull/94502

Reply:
Also, if you have multiple comments for a PR, you can use the "Start Review" button, so that all your comments fall under one review. Otherwise, each of the review messages generates individual emails and become separate reviews @gabrielcaires